base-template/about-base-template.md

```diff
@@ -11,84 +24,349 @@ The base template goes one step further, in that it helps you create any templat
+* Technical writing: To produce a well-written document that includes the relevant references.
+
+## Target audience for templates
+
+The templates target multiple audiences. They include writing tips for the following roles:
+
+
+### Template author
+
+{Template author tip: Remove this “template author” section from about-{doctype}-template.md files.}
+
+A template author creates a new template in line with base and core templates.
+
+
+### Template reviewer
+
+{Template author tip: Remove this “template reviewer” section from about-{doctype}-template.md files.}
+
+A template reviewer reviews the new template against template guidelines.
+
+
+### Template customizer
+
+A template customizer customizes a doctype template to align with a specific project or organizational guidelines. For example, specify the style guide to use with the template.
+
+
+### Docset owner
+
+A docset owner establishes an Information Architecture and Documentation Management Plan for a project. Typical tasks include:
+
+* Select a set of doctype templates for the project.
+* Customize templates if needed.
+* Select a style guide for the project.
+* Establish a docset maintenance processes.
+
+A docset owner should capture documentation decisions within a Documentation Management Plan.
+
+_{TBD: We should define a template for a Documentation Management Plan, then reference from here.}_
+
+The [Information Architecture Guide](https://thegooddocsproject.dev/ia-guide.html) aims to help docset owners with some of these tasks.
+
+### Document author
+
+A document author writes content in line with templates. Typically the document author is a subject matter expert and not a dedicated technical writer.
+
+Ideally, their document will receive feedback and guidance from a document reviewer and/or their docset owner.
+
+### Document reviewer
+
+Documentation is almost always improved when it is reviewed against the doctype’s template, the selected style guide, writing best practices and is reviewed for technical accuracy. While not essential, it helps if one of the reviewers is a dedicated technical writer.
+
+### Document reader
+
+And of course, we need to understand the needs of the final document reader. There are likely multiple reader personas to consider, which depends upon the reader’s role, their technical expertise, their specific goal, and so on.
+
+## Inheritance of template sets
+
+There is a hierarchy of inheritance between template documents.
+
+_{TBD: Insert hierarchy diagram here.}_
+
+### Base (abstract) template set
+
+The base template describes common features which are to be inherited by all doctype templates.
+
+**Target audience:** Template authors.
+
+**Documents:**
+
+* base-template.md: Provides common structure, elements and writing tips for downstream xxx-template.md files.
+* about-base-template.md: Provides common structure, elements and writing tips for downstream about-xxx-template.md files.
+
+### Core (abstract) templates set
```

Comment:
+ for Core templates set description.

Reply:
Hi @viraji , I'm unclear if you are suggesting a specific action here? If so, I'm unclear on what needs to change.
I'm assuming you are suggesting a +1, as per some of your other comments, and will close no action.
Please re-introduce your comment (with a specific action) if I've missed your point.
Thanks.